addons/project/static/src/views/project_task_kanban/project_task_kanban_column_quick_create.js

```diff
@@ -0,0 +1,9 @@
+/** @odoo-module **/
+
+import { KanbanColumnQuickCreate } from "@web/views/kanban/kanban_column_quick_create";
+
+export class ProjectTaskKanbanColumnQuickCreate extends KanbanColumnQuickCreate {
+    canShowExamples() {
+        return super.canShowExamples() && this.props.groupByFieldString === 'Stage';
+    }
+}
```

Comment:
This won't be necessary anymore.
Just add `allowedGroupBys: ["stage_id"]` in your kanban examples registry entry.

See recent changes in (in a standalone fix commit): #115909

Reply:
In fact you don't have to do something, the `allowedGroupBys` key will already be set :-)